playground-nextjs/next.config.js

```diff
@@ -1,6 +1,6 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  /* config options here */
-};
+	/* config options here */
```

Comment:
Since we are fixing the formatting let's go further and remove the gigantic indentation on the files. I did already on a separate refactor PR but i will leave to your call @ryami333 if you want to add it to the PR.

Steps: 
 -Open root `package.json`
 - Go down to the property `prettier` and remove `useTabs`
 - Run `npm run prettier` again

The PR files will increase to around ~44 but since they are only identation changes its gonna be easier to check.

![Screenshot 2024-07-08 at 10 45 38](https://github.com/storyblok/storyblok-js-client/assets/4699008/e82c5041-20e0-4274-877c-154d64eaf6ad)

Reply:
Done ✅

Reply:
I misinterpreted the request initially - I inadvertently removed the `trailingComma` rule as well. I've just re-instated that and pushed.